### PR TITLE
3665 - More verbose error messages.

### PR DESCRIPTION
**Pull request by sf-dlund**
Updated errors.py to show more verbose error messages.

### CODE REVIEW

File: fs/errors.py [hunk 1]
```diff
@@ -83,128 +84,139 @@ class CreateFailedError(FSError):
...
 
 class UnsupportedError(OperationFailedError):
     """Exception raised for operations that are not supported by the FS."""
-    default_message = "Unable to %(opname)s: not supported by this filesystem"
+    default_message = "Unable to %(opname)s (original_exc = %(originalExc)s): not supported by this filesystem [%(details)s]"
 
 
 class RemoteConnectionError(OperationFailedError):
     """Exception raised when operations encounter remote connection trouble."""
+<<<<<<< HEAD
```
Comment (galitskyd) [inline fs/errors.py]: left over merge conflict

**Comment by travcunn** [comment]
See my comment on user-friendly messages in the Assembla ticket

**Comment by giampaolo** [comment]
With this I think also the frontend will be impacted as it's gonna show the original low-level exception on screen. I would just store the exception in a class attribute but not modify `__str__` as the web ui relies on it all over the place. If we'll ever switch to Python 3 this problem will be solved for free thanks to chained exceptions which are built-in in the language.

**Comment by sf-dlund** [comment]
Branch updated to remove original exception from messages but retain within FSError class.

**Comment by galitskyd** [comment]
@giampaolo @travcunn How does this look now?

File: fs/errors.py [hunk 2]
```diff
@@ -49,30 +49,31 @@ class FSError(Exception):
     """Base exception class for the FS module."""
     default_message = "Unspecified error"
 
-    def __init__(self,msg=None,details=None):
+    def __init__(self, msg=None, details=None, originalExc=None):
```
Comment (giampaolo) [inline fs/errors.py]: I would follow the existing naming convention and use `original_exc` instead

File: fs/errors.py [hunk 3]
```diff
@@ -85,231 +86,264 @@ class PathError(FSError):
     """
     default_message = "Path is invalid: %(path)s"
 
-    def __init__(self,path="",**kwds):
+    def __init__(self, path="", originalExc=None**kwds):
```
Comment (giampaolo) [inline fs/errors.py]: Missing comma? Did tests run?

**Pull request merged**